openapi/platform/platform.yaml

```diff
@@ -4441,7 +4441,9 @@ paths:
       description: |-
         [Status](https://developers.vendasta.com/platform/ZG9jOjEwMTU2NTYy-versioning): `Proposed`
 
-        Provides a list of products that can be activated by a partner for their customers. You must filter the list to a single partner using the `filter[partner.id]=PID` query param.
+        Provides a list of products that can be activated by a partner for their customers. 
+        You must filter the list to a single partner using the `filter[partner.id]=PID` query param. 
+        Sorted by the whiteLabelName field in ascending order.
```

Comment:
Are you sure that it is the white label name and not the vendor's name? That is great if that is correct, I'm just surprised.

Reply:
Yes. The internal API currently provides only the Partner given white label name for a product (if white labelling is done). We are retrieving vendor name from a different API.